```
04/12/26-22:37:21 DEBUG KoboPlugin: Cache hit for virtual book: KOBO_VIRTUAL://0NWZ215MS9J5P/Amor Towles - A Gentleman in Moscow.epub
04/12/26-22:37:21 DEBUG ImageWidget: _render'ing data 310 477
04/12/26-22:37:21 DEBUG ImageWidget: scaling by 0.21693491952414
04/12/26-22:37:21 DEBUG RenderImage:scaleBlitBuffer: scaling
bash: line 1: 268108 Segmentation fault         (core dumped) EMULATE_READER_W=1080 EMULATE_READER_H=720 EMULATE_READER_DPI= ./luajit reader.lua "$@"
make: *** [make/emulator.mk:20: run] Error 139
``` 

😱 lovely

--- a/src/virtual_library.lua
+++ b/src/virtual_library.lua
@@ -252,7 +252,7 @@ function VirtualLibrary:getMetadataForPath(path, include_cover)
     end
 
     result.has_cover = "Y"
-    result.cover_bb = cover_bb
+    result.cover_bb = cover_bb:copy()
     result.cover_w = cover_bb:getWidth()
     result.cover_h = cover_bb:getHeight()
     result.cover_sizetag = string.format("%dx%d", result.cover_w, result.cover_h)

